invokeai/backend/stable_diffusion/diffusers_pipeline.py

```diff
@@ -168,39 +169,28 @@ def __init__(
         self.invokeai_diffuser = InvokeAIDiffuserComponent(self.unet, self._unet_forward)
 
     def _adjust_memory_efficient_attention(self, latents: torch.Tensor):
-        """
-        if xformers is available, use it, otherwise use sliced attention.
-        """
         config = get_config()
-        if config.attention_type == "xformers":
-            self.enable_xformers_memory_efficient_attention()
-            return
-        elif config.attention_type == "sliced":
+        attention_type = config.attention_type
+        if attention_type in ["normal", "xformers"]:
```

Comment:
This warning and deprecation logic is repeated verbatim in line 59 of `custom_attention.py`. I suggest turning it into a common utility function.

Reply:
Yes, but this part intended to be deleted a bit later in modular backend, so I not sure that we really need do this.
I only patched logic according to removing attention types.